docbrown/db/src/graphdb.rs

```diff
@@ -704,7 +704,7 @@ mod db_tests {
     fn db_lotr() {
         let g = GraphDB::new(4);
 
-        let data_dir: PathBuf = [env!("CARGO_MANIFEST_DIR"), "resources/test/lotr.csv"]
+        let data_dir: PathBuf = [env!("CARGO_MANIFEST_DIR"), "resources/withheader/lotr.csv"]
```

Comment:
We probably don't need to create separate directories for files of each type. Just name them accordingly. For instance `lotr-with-header.csv`